dev/io.openliberty.data.internal.persistence/src/io/openliberty/data/internal/persistence/RepositoryImpl.java

```diff
@@ -278,7 +279,7 @@ private QueryInfo completeQueryInfo(EntityInfo entityInfo, QueryInfo queryInfo)
 
         Method method = queryInfo.method;
         Class<?> multiType = queryInfo.getMultipleResultType();
-        boolean countPages = Page.class.equals(multiType) || KeysetAwarePage.class.equals(multiType);
+        boolean countPages = Page.class.equals(multiType) || CursoredPage.class.equals(multiType) || KeysetAwarePage.class.equals(multiType);
```

Comment:
```suggestion
        boolean countPages = Page.class.equals(multiType) || CursoredPage.class.equals(multiType);
```
Is `KeysetAwarePage` still needed here, since it's replaced by `CursoredPage`?

Reply:
I see there's a TODO below this, so is this still required temporarily?

Reply:
Yes, we are keeping the old interfaces working until after the cutoff for the beta so it can comply with Jakarta Data M3.